server/server.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var path = require('path');
+var bodyParser = require('body-parser');
+
+var app = express();
+
+app.use(bodyParser.json());
+
+app.get('/card', (req, res) => {
```

Comment:
For sure that would make more sense I was wondering if we should redirect '/' to '/card' but that solves the problem!